examples/python/secure_grpc_client.py

```diff
@@ -0,0 +1,59 @@
+import grpc
+import argparse
+from armonik.client.sessions import ArmoniKSessions, SessionFieldFilter
+from armonik.common.enumwrapper import SESSION_STATUS_CANCELLED
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description="ArmoniK Example for Client connection TLS or mutual TLS")
+    parser.add_argument("-v","--version", action="version", version="ArmoniK Admin CLI 0.0.1")
+    parser.add_argument("--endpoint", default="localhost:5001", help="ArmoniK control plane endpoint")
+    parser.add_argument("--ca", help="ca.crt path for TLS or mutual TLS")
+    parser.add_argument("--cert", help="client.submitter.crt for mutual TLS")
+    parser.add_argument("--key", help="client.submitter.key for mutual TLS")
+    return parser.parse_args()
+
+def create_channel(endpoint: str,  ca: str = None, key: str = None, cert: str = None) -> grpc.Channel:
+    """
+    Create a gRPC channel for communication with the ArmoniK control plane
+
+    Args:
+        ca (str): CA file path for TLS or mutual TLS
+        cert (str): Certificate file path for mutual TLS
+        key (str): Private key file path for mutual TLS
+        endpoint (str): ArmoniK control plane endpoint
+
+    Returns:
+        grpc.Channel: gRPC channel for communication
+    """
+    if ca != None:
```

Comment:
The CA is only mandatory if you've not installed it on your system. cf https://ubuntu.com/server/docs/security-trust-store

What you should test for is whether the endpoint is called using http or https (or if it doesn't contain the scheme, then by a flag (-ssl for example))
In the case you want simple tls without a ca, you can construct a default empty grpc.ssl_channel_credentials